bin/mocha

```diff
@@ -9,7 +9,7 @@
 
 const spawn = require('child_process').spawn;
 const path = require('path');
-const getOptions = require('./options');
+const getOptions = require('./options').getOptions;
```

Comment:
I should not have changed this file, right?
But I had to, since I adapted the module.exports object of options.js